dart/lib/src/hub.dart

```diff
@@ -312,6 +321,11 @@ class Hub {
       }
     }
   }
+
+  /// add an integration disposer
+  /// disposers are called on [Hub.close]
+  void addIntegrationDisposer(IntegrationDisposer disposer) =>
+      _integrationDisposers.add(disposer);
```

Comment:
why does it need to be a public method on the Hub, can we get rid of it somehow?

Reply:
if we can't do it fully automatic, I'd rather prefer to add as named/positional parameter to the  `options.addIntegration` method